In [1]:
import numpy as np

### Урок 5

#### 1. Известно, что генеральная совокупность распределена нормально со средним квадратическим отклонением, равным 16. Найти доверительный интервал для оценки математического ожидания a с надежностью 0.95, если выборочная средняя M = 80, а объем выборки n = 256.

Границы доверительного интервала находим по формуле:
$$T_{1,2} = \overline{X} \pm Z_{\frac{\alpha}{2}}\cdot \frac{\sigma}{\sqrt{n}}$$
По таблице находим $Z_{\frac{\alpha}{2}}=1.96$<br>
По условию задачи $\overline{X}=80, \ \sigma=16, \ n=256$<br>
Находим границы доверительного интервала:
$$T_{1,2} = 80 \pm 1.96\cdot \frac{16}{\sqrt{256}}$$
$$T_{1,2} = [78.04;81.96]$$

#### 2. В результате 10 независимых измерений некоторой величины X, выполненных с одинаковой точностью, получены опытные данные: 6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1 Предполагая, что результаты измерений подчинены нормальному закону распределения вероятностей, оценить истинное значение величины X при помощи доверительного интервала, покрывающего это значение с доверительной вероятностью 0,95. 

Границы доверительного интервала находим по формуле:
$$T_{1,2} = \overline{X} \pm t_{\frac{\alpha}{2}}\cdot \frac{\sigma}{\sqrt{n}}$$
По таблице находим $t_{\frac{\alpha}{2}}=2.262$<br>
Находим $\overline{X}$ по выборке:

In [11]:
a = (6.9, 6.1, 6.2, 6.8, 7.5, 6.3, 6.4, 6.9, 6.7, 6.1)
x = sum(a) / len(a)
x

6.589999999999999

Находим $\sigma_{н}$ по выборке:

In [13]:
sigma = np.std(a, ddof=1)
sigma

0.4508017549014448

Находим границы доверительного интервала:
$$T_{1,2} = 6,59 \pm 2,262\cdot \frac{0,45}{\sqrt{10}}$$
$$T_{1,2} = [6,2681;6,9119]$$

#### 3. Утверждается, что шарики для подшипников, изготовленные автоматическим станком, имеют средний диаметр 17 мм. Используя односторонний критерий с α=0,05, проверить эту гипотезу, если в выборке из n=100 шариков средний диаметр оказался равным 17.5 мм, а дисперсия известна и равна 4 кв.мм.

**1)**<br>
$H_0:диаметр=17 мм, \mu=\mu_0$<br>
$H_1:диаметр=17,5 мм, \mu>\mu_0$<br>
**2)**<br>
Критерий - Z<br>
**3)**<br>
Уровень значимости $\alpha=0.05 \ (5\%)$<br>
**4)**<br>
$Z_{Табличное}=1.645, \ $ для уровня значимости $\alpha=5\%$<br>
$Z_{Расчетное}=\frac{\overline{X}-\mu_0}{\frac{\sigma}{\sqrt{n}}}=(17.5-17)\cdot \frac{5}{1}=2.5$<br>
**5)**<br>
$H_1 \ $верна для уровня значимости $\alpha=5\%$

#### 4. Продавец утверждает, что средний вес пачки печенья составляет 200 г. Из партии извлечена выборка из 10 пачек. Вес каждой пачки составляет: 202, 203, 199, 197, 195, 201, 200, 204, 194, 190. Известно, что их веса распределены нормально. Верно ли утверждение продавца, если учитывать, что доверительная вероятность равна 99%?

$$t_{Расчетное}=\frac{\overline{X}-\mu}{\frac{\sigma}{\sqrt{n}}}=\frac{\overline{X}-\mu}{\sigma}\cdot \sqrt{n}$$
Находим $\overline{X}$ по выборке:

In [14]:
a = (202, 203, 199, 197, 195, 201, 200, 204, 194, 190)
x = sum(a) / len(a)
x

198.5

Находим $\sigma_{н}$ по выборке:

In [15]:
sigma = np.std(a, ddof=1)
sigma

4.453463071962462

**1)**<br>
$H_0:\mu=200 г$<br>
$H_1:\mu\ne200 г$<br>
**2)**<br>
Критерий - t<br>
**3)**<br>
Уровень значимости $\alpha=0.01 \ (1\%)$<br>
**4)**<br>
По таблице критических точек распределения Стьюдента находим критическую точку для
уровня значимости α = 0,01 и числу степеней свободы v = 9 : $$t_{кр}≈ 3,25$$
$t_{Расчетное}=\frac{\overline{X}-\mu}{\sigma}\cdot \sqrt{n}=\frac{198.5-200}{4.4535}\cdot \sqrt{10}\approx-1.065$<br>
**5)**<br>
Так как $\left|t_{Расчетное}\right| = 1.065 < t_{кр}=3,25 $, то $H_0$ гипотезу о равенстве среднего веса 200 г
можно принять.